sign-action.md

```diff
@@ -0,0 +1,159 @@
+# Sign an image with Notation in GitHub Actions
+
+This document walks you through how to create a GitHub Actions workflow to achieve the following goals:
+
+1. Build an image and push it to Azure Container Registry (ACR)
+2. Sign the image with Notation and Notation AKV plugin with a signing key stored in Azure Key Vault (AKV). The generated signature is automatically pushed to ACR
+
+## Prerequisites
+
+- You have created a Key Vault in AKV and created a self-signed signing key and certificate. You can follow this [doc](https://learn.microsoft.com/en-us/azure/container-registry/container-registry-tutorial-sign-build-push#create-a-self-signed-certificate-azure-cli) to create self-signed key and certificate for testing purposes. 
+- You have created a registry in Azure Container Registry
+- You have a GitHub repository to store the sample workflow file and GitHub Secrets
+
+## Create GitHub Secrets to store credentials
+
+Enter your GitHub repository, create two encrypted secrets `ACR_PASSWORD` and `AZURE_CREDENTIALS` for the repository to authenticate with ACR and AKV. See the [GitHub doc](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) for details.
```

Comment:
By reading this, I think we still need a brief definition of these two terms. Btw, we need to write out what the `GitHub doc` link is about, otherwise, it will likely be skipped by the reader.
```suggestion
Enter your GitHub repository, create two encrypted secrets `ACR_PASSWORD` and `AZURE_CREDENTIALS` in the repository to authenticate with ACR and AKV. See [creating encrypted secrets for a repository](https://docs.github.com/en/actions/security-guides/encrypted-secrets#creating-encrypted-secrets-for-a-repository) for details.

- `ACR_PASSWORD`: the password of your ACR registry where your image will be stored.
- `AZURE_CREDENTIALS`: the credential to access your AKV where your signing key pair is stored.
```